core/src/com/mygdx/game/Global.java

```diff
@@ -0,0 +1,8 @@
+
+package com.mygdx.game;
+
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+
+public class Global {
+	public static SpriteBatch batch;
```

Comment:
I didn't have so clear what the `SpriteBatch` was now it makes that as a drawing tool it is costly to instantiate it multiple times